### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by mjneil**
README is quite out of date. Here are some updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
 playlist. Once it has been retreived, it will become the active media
 playlist.
 
-#### hls.segmentXhrTime
+#### hls.systemBandwidth
 Type: `number`
 
-The number of milliseconds it took to download the last media segment.
-This value is updated after each segment download completes.
+`systemBandwidth` is a combination of two serial processes bit-rates. The first
```
Comment (OshinKaramian) [inline README.md]: Gonna be super annoying: I think the grammar here is processes', given that it's plural/possessive.

Comment (mjneil) [reply README.md]: I encourage you to tear apart my grammar

File: README.md [hunk 2]
```diff
@@ -329,7 +331,7 @@ Runtime properties are attached to the tech object when HLS is in
 use. You can get a reference to the HLS source handler like this:
 
 ```javascript
-var hls = player.tech({ IWillNotUseThisInPlugins: true }).hls;
+var hls = player.tech().hls;
```
Comment (gesinger) [inline README.md]: Should we change this and the other `hls` references to `vhs`?

File: README.md (hunk 1 above)
Comment (gesinger) [reply README.md]: Also we can just do bitrates without the dash, since it is used elsewhere

File: README.md [hunk 3]
```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
...
 
-#### hls.segmentXhrTime
+#### hls.systemBandwidth
 Type: `number`
 
-The number of milliseconds it took to download the last media segment.
-This value is updated after each segment download completes.
+`systemBandwidth` is a combination of two serial processes bit-rates. The first
+is the network bitrate provided by `bandwidth` and the second is the bitrate of
+the entire process after that - decryption, transmuxing, and appending - provided
```
Comment (gesinger) [inline README.md]: May be easier to do a parenthetical

**Comment by OshinKaramian** [comment]
Once this is pulled up, should we carry these changes over to videojs-contrib-hls as well?

File: README.md [hunk 4]
```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
...
 have a more accurate source of bandwidth information, you can override
 this value as soon as the HLS tech has loaded to provide an initial
 bandwidth estimate.
 
-#### hls.bytesReceived
+#### hls.throughput
 Type: `number`
 
-The total number of content bytes downloaded by the HLS tech.
+The number of bits decrypted, transmuxed, and appended per second as a cumulative average.
```
Comment (gesinger) [inline README.md]: This reads a bit like it averages across all time, not just processing time. Might be worth it to add a clarification saying something like `(only averages across active processing time)`

File: README.md [hunk 5]
```diff
@@ -485,6 +490,33 @@ player.ready(function() {
...
+| buffered              | array  | List of time ranges of content that are in the SourceBuffer |
+| currentTime           | number | The current position of the player |
+| currentSource         | object | The source object. Has the structure `{src: 'url', type: 'mimetype'}` |
+| currentTech           | string | The name of the tech in use |
+| duration              | number | Duration of the video in seconds |
+| master                | object | The [master playlist object](#hlsplaylistsmaster) |
+| playerDimensions      | object | Contains the width and height of the player |
+| seekable              | array  | List of time ranges that the player can seek to |
+| timestamp             | number | Timestamp of when `hls.stats` was accessed |
+| videoPlaybackQuality  | object | Media playback quality metrics as specified by the W3C's Media Playback Quality API |
```
Comment (gesinger) [inline README.md]: Not a big deal, but could link

File: README.md [hunk 6]
```diff
@@ -638,11 +670,25 @@ you attempt to play an HLS stream with fragmented MP4 segments, Edge
...
 
+## Debugging
+
+videojs-http-streaming makes use of `videojs.log` for debug logging. You can enable these logs
+by setting the log level to `debug` using `videojs.log.level('debug')`. You can access a complete
+history of the logs using `videojs.log.history()`. This history is maintained even when the
+log level is not set to `debug`.
+
+`hls.stats` can also be helpful when debugging. Accessing this object will give you
+a snapshot summary of various HLS and player stats. See [hls.stats](#hlsstats) for details
```
Comment (gesinger) [inline README.md]: Maybe VHS instead of HLS?

File: README.md (hunk 3 above)
Comment (mjneil) [reply README.md]: `(decryption, transmuxing, and appending)` ?

File: README.md (hunk 2 above)
Comment (mjneil) [reply README.md]: We haven't updated all references within the code to have `vhs` yet, e.g. `player.tech().vhs` is `undefined` at the moment

**Pull request merged**